docs/events/mouse_click.md

```diff
@@ -0,0 +1,20 @@
+# Click
+
+The `Click` event is sent to a widget when the user clicks a mouse button.
+
+- [x] Bubbles
+
+## Attributes
+
+| attribute  | type | purpose                                   |
+| ---------- | ---- | ----------------------------------------- |
+| `x`        | int  | Mouse x coordinate, relative to Widget    |
+| `y`        | int  | Mouse y coordinate, relative to Widget    |
+| `delta_x`  | int  | Change in x since last mouse event        |
+| `delta_y`  | int  | Change in x since last mouse event        |
+| `button`   | int  | Index of mouse button                     |
+| `shift`    | bool | Shift key pressed if True                 |
+| `meta`     | bool | Meta key pressed if True                  |
+| `ctrl`     | bool | Shift key pressed if True                 |
```

Comment:
I'd find it clearer if it had been phrased the other way around:

```suggestion
| `shift`    | bool | True if Shift key pressed                 |
| `meta`     | bool | True if Meta key pressed                  |
| `ctrl`     | bool | True if Shift key pressed                 |
```

Reply:
Me too :)